### ISSUE TITLE
What is the currant state of this plugin

### ISSUE THREAD

**Issue by woakes070048** [nathando/erpnext-opencart-api#1]
What is the current state of this plugin. does it need more development and has it been abandoned

**Comment by nathando** [comment]
sorry this repo has been abandoned. I am currently not actively developing on ERPNext anymore. You can check out the forks to see if there are ones which are more active. Thanks!

**Comment by woakes070048** [comment]
Was it in a working state. I don't mind finishing it

**Comment by nathando** [comment]
Sorry overlooked it. Can you check this fork ? https://github.com/olhonko/erpnext-opencart-api/ This is more well maintained.

**Comment by woakes070048** [comment]
@olhonko are you still working on this?

**Comment by olhonko** [comment]
@woakes070048 yes, I'm still working on this, but mostly on supporting people that already using this plugin on production.

**Comment by woakes070048** [comment]
Okay i was planning useing it for a project. does it support opencart 2?

**Comment by olhonko** [comment]
@woakes070048 yes, but it requires REST API Admin plugins (https://opencart-api.com) to be installed.
BTW, my fork of erpnext-opencart-api is not too generic.

**Comment by woakes070048** [comment]
@olhonko what do you mean by not to generic?

**Comment by olhonko** [comment]
@woakes070048 it's adapted for one of the production erpnext system.

**Comment by woakes070048** [comment]
@olhonko I am planning on using erpnext 7 and opencart 2 so i will let you know how it goes. Should I use your branch or @nathando?

**Comment by zenny** [comment]
@olhonko, @nathando referred (https://github.com/nathando/erpnext-opencart-api/issues/1#issuecomment-217721576) to your repo at https://github.com/olhonko/erpnext-opencart-api/ as an updated one, but the link seems to be dead. Or is the project really dead?! 

Cheers,
